tools/bqman/golang/bqman_test.go

```diff
@@ -0,0 +1,269 @@
+package main
```

Comment:
This file is missing the licensing header

Reply:
Done